src/pheval/cli_pheval_utils.py

```diff
@@ -27,3 +34,77 @@ def scramble_semsim():
 def scramble_phenopacket():
     """scramble_phenopacket"""
     print("running pheval_utils::scramble_phenopacket command")
+
+
+@click.command()
+@click.option(
+    "--left",
+    "-l",
+    required=True,
+    metavar="FILE",
+    help="Path of first file that will be compared",
+)
+@click.option(
+    "--right",
+    "-r",
+    required=True,
+    metavar="FILE",
+    help="Path of second file that will be compared",
+)
+@click.option(
+    "--join-columns",
+    "-j",
+    required=False,
+    multiple=True,
+    default=["GENE_SYMBOL"],
+    metavar="LIST",
+    help="""Represents the parameter that specifies which
+columns will be used to join left and right inputs.""",
```

Comment:
Provide a better description here, maybe something along the lines of "primary key".